src/ess/offspec/__init__.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+import sciline
+
+from ..reflectometry import providers as reflectometry_providers
+from . import conversions, data, load, maskings, normalization, types, workflow
+
+providers = (
+    *reflectometry_providers,
+    *load.providers,
+    *conversions.providers,
+    *maskings.providers,
+    *workflow.providers,
+    *normalization.providers,
+)
+
+
+def OffspecWorkflow() -> sciline.Pipeline:
```

Comment:
Please move this into `workflow.py`. There should be no definitions in `__init__.py`, only imports.